CMakeLists.txt

```diff
@@ -47,6 +47,10 @@ else()
   endif()
 endif()
 
+if (SHADOW_FORBIDDEN_FUNCTIONS)
```

Comment:
Should this be defined as a CMake option explicitly?

Reply:
Unsure, I think it's slightly better, but we are not very consistent on this.

Can I send that in a follow up? I think it's still functional without.

Reply:
Done in the follow up at https://github.com/duckdb/duckdb/pull/17894